builtin/game/misc.lua

```diff
@@ -234,8 +234,14 @@ else
 	end
 
 	function core.colorize(color, message)
-		return core.get_color_escape_sequence(color) .. message .. core.get_color_escape_sequence("#ffffff")
+		local output = ""
+		for line in message:gmatch("([^\n]+)") do
```

Comment:
This could be better done as:

```
local out, n = { }, 0
local col = core.get_color_escape_sequence(color)
local def = core.get_color_escape_sequence("#ffffff")
for line in message:split("\n") do
  n = n + 1
  out[n] = col..line..def
end
return table.concat(t, "\n")
```